platforms/documentation/docs/src/samples/writing-tasks/custom-test-task/tests/failing.out

```diff
@@ -0,0 +1,8 @@
+test > FailingSuite > failingTest() FAILED
+    org.gradle.api.tasks.VerificationException: This is a test failure
+
+FAILURE: Build failed with an exception.
+
+* What went wrong:
+Execution failed for task ':test'.
```

Comment:
💭 Nothing to be done about it here, but if you're writing and debugging a custom task, then seeing "Execution failed for task ':test'" at all is really confusing/demotivating.  The task works just fine, it's the test that failed.